operations/research-and-development/product/growth/ab-testing.md

```diff
@@ -0,0 +1,119 @@
+# Overview
+
+This document will provide an overview of Mattermost’s A/B Testing methodology. It consists of a sequence of steps and considerations to design valid A/B tests.
+
+## A/B Testing Methodology
+
+### Success Criteria
```

Comment:
Before the success criteria, I'd recommend they start by defining the goal or objective. E.g. is the goal to increase user retention? Free-to-paid conversion? Something else?

Reply:
The success criteria is essentially synonymous with the goal or objective, but I see what you mean. Define the purpose of the experiment, which then allows you to determine/define the success criteria.

Reply:
Yeah, exactly. For instance, using our invite user test example, the success criteria is number of unique users inviting people to the system, with an objective of increasing early user retention.

However, if the objective was to increase free-to-paid conversion, then the success criteria may be something different, such as inviting their manager or a decision-maker into the instance.